kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -224,10 +224,17 @@ void addHandlers(Channel ch, EndpointBinding binding) {
 
         pipeline.addLast("responseEncoder", new KafkaResponseEncoder());
 
-        var clientToProxyCounterProvider = Metrics.KROXYLICIOUS_CLIENT_TO_PROXY_REQUEST_TOTAL_METER_PROVIDER.create(virtualCluster.getClusterName(), binding.nodeId());
-        var proxyToClientCounterProvider = Metrics.KROXYLICIOUS_PROXY_TO_CLIENT_RESPONSE_TOTAL_METER_PROVIDER.create(virtualCluster.getClusterName(), binding.nodeId());
-
-        pipeline.addLast("downstreamMetrics", new MessageMetrics(clientToProxyCounterProvider, proxyToClientCounterProvider));
+        var clientToProxyMessageCounterProvider = Metrics.KROXYLICIOUS_CLIENT_TO_PROXY_REQUEST_TOTAL_METER_PROVIDER.create(virtualCluster.getClusterName(),
+                binding.nodeId());
+        var clientToProxyMessageSizeDistributionProvider = Metrics.KROXYLICIOUS_CLIENT_TO_PROXY_REQUEST_SIZE_METER_PROVIDER.create(virtualCluster.getClusterName(),
+                binding.nodeId());
+        var proxyToClientMessageCounterProvider = Metrics.KROXYLICIOUS_PROXY_TO_CLIENT_RESPONSE_TOTAL_METER_PROVIDER.create(virtualCluster.getClusterName(),
+                binding.nodeId());
+        var proxyToClientMessageSizeDistributionProvider = Metrics.KROXYLICIOUS_PROXY_TO_CLIENT_RESPONSE_SIZE_METER_PROVIDER.create(virtualCluster.getClusterName(),
+                binding.nodeId());
```

Comment:
I think the pattern established with the `Metrics.ClusterNodeSpecificMetricProviderCreator` type and constants of that type in `Metrics` is a bit weird. We never seem to actually pass around a value of type `ClusterNodeSpecificMetricProviderCreator`. Instead the constants values are always immediately used to call `create()`. So AFAICS we can just replace each constant with a static method. 

This would avoid the (frankly ridiculously) long names we're using here, e.g. 
```java
// decl
Metrics {
  static DistributionSummary proxyToClientResponseSize(String clusterName, int nodeId) { ... }
}
// callsite
var proxyToClientMessageSizeDistributionProvider = Metrics.proxyToClientResponseSize(virtualCluster.getClusterName(), binding.nodeId())
```

Reply:
Thats a point I failed to raise on the last metrics PR, why do the constant names include the full meter name? The `Kroxylicious` prefix is only really needed for external disambiguation purposes. Internally we can just use `PROXY_TO_CLIENT_RESPONSE_SIZE`